NorthstarLauncher/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ target_link_libraries(NorthstarLauncher PRIVATE
                       uuid.lib
                       odbc32.lib
                       odbccp32.lib
-                      Ws2_32.lib
+                      ws2_32.lib
```

Comment:
Shouldn't this be `WS2_32`?

Reply:
Yep, information available was incorrect, will use the correct naming.